clevertap-core/src/test/java/com/clevertap/android/sdk/CleverTapAPITest.kt

```diff
@@ -605,6 +599,188 @@ class CleverTapAPITest : BaseTestCase() {
             }
         }
     }
+
+    @Test
+    fun `test getUserEventLogCount`(){
+        // Arrange
+        val evt = "test"
+        `when`(corestate.localDataStore.readUserEventLogCount(evt)).thenReturn(1)
+        
+        // Act
+        mockStatic(CTExecutorFactory::class.java).use {
+            `when`(CTExecutorFactory.executors(any())).thenReturn(
+                MockCTExecutors(
+                    cleverTapInstanceConfig
+                )
+            )
+            mockStatic(CleverTapFactory::class.java).use {
+                `when`(CleverTapFactory.getCoreState(application, cleverTapInstanceConfig, null))
+                    .thenReturn(corestate)
+                cleverTapAPI = CleverTapAPI.instanceWithConfig(application, cleverTapInstanceConfig)
```

Comment:
this is common block in all test cases, can we extract?

Reply:
fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/cde4f26382c4e1d147db3583d528dd34b6ac0167